workers/ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -125,7 +125,7 @@ def calculate(self) -> None:
             median_diff = "in the last {0} years".format(median_diff)
         self.result.description = Template(self.metadata.result_description).substitute(
             years=median_diff,
-            layer_name=self.layer.name,
+            layer_key=self.layer.name,
```

Comment:
Shouldn't it be `key`?

Reply:
Actually it should not be the key but the name here since it is formated text.
I changed it back to the layer name (also in the html template)